packages/terra-popup/tests/nightwatch/popup-spec.js

```diff
@@ -0,0 +1,233 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-Popup', done);
+  },
+
+  'isOpen prop - RESIZE': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/popup-tests/default`)
+      .waitForElementPresent('#default-button', 1000)
```

Comment:
There would still be tests to ensure the element is present on the page like [here](https://github.com/cerner/terra-core/pull/623/files#diff-54e3ca03d43643bd79a2bae13b4f9cb3R17) if no events are occurring. If we do have events occurring, like this your test here with the button click, we have something like [this](https://github.com/cerner/terra-clinical/pull/151/files#diff-a71d5e5423e814bff432415bc9f88fabR107). So each `.waitForElementPresent` in your tests could be changed to `.assert.elementPresent`.